scripts/policy/frameworks/dpd/packet-segment-logging.zeek

```diff
@@ -4,7 +4,7 @@
 ##! A caveat to logging packet data is that in some cases, the packet may
 ##! not be the packet that actually caused the protocol violation.
 
-@load base/frameworks/dpd
+@load base/frameworks/analyzer
```

Comment:
This needed?

Reply:
Dropped. The policy files are suspiciously in `policy/frameworks/dpd` still, but there's not `policy/frameworks/analyzers`, so keeping it as is.